### DISCUSSION TITLE
Dynamically-generated content with Agate?

### DISCUSSION THREAD

**Discussion by d47081** [mbrubeck/agate#328] [Q&A] [answered]
Thoughts to write simple app with content, that generated on back-end, it's possible to pass the `.gmi` to PHP or another processor?

**Comment by d47081** [comment]
It seems that in gemini usage model, I must to write/fork own server for each custom app. 

Bit new after classical web :)

**Comment by Johann150** [comment, marked as answer]
Agate is a "very simple server for the Gemini hypertext protocol" and not intended to be used for dynamic content.

see also the pinned issue #35

**Reply by d47081** [reply]
Thank you, clear
